ui/src/reportAnIssue.ts

```diff
@@ -14,19 +14,35 @@ import { getPackageInfo } from "./getPackageInfo";
 export function reportAnIssue(actionId: string, parsedError: IParsedError): void {
     const { extensionName, extensionVersion } = getPackageInfo();
 
-    const body: string = `
-&lt;Please be sure to remove any private information before submitting.&gt;
+    let body: string = `
```

Comment:
In mine, I added the request to fill out repro steps right to the top to hopefully encourage them more and give them less excuse. :-)  I'll let you decide.

<!-- Please fill out the repro steps to help us better understand your issue, and be sure to remove any private information before submitting. -->

Reply:
Added #423

Reply:
Not entirely sure what you mean and I can't find your old commit at this point so I'll merge as-is and you can edit if you want